indexer/cmd/indexer-refresh/main.go

```diff
@@ -37,7 +47,7 @@ func main() {
 		"by account on L1 and L2"
 
 	app.Action = indexer.Main(GitVersion)
-	err := app.Run(os.Args)
+	err = app.Run(os.Args)
```

Comment:
could do something like:

```golang
if err := app.Run(os.Args); err != nil {
  log.Crit("...")
}
```

to keep the second error only in scope of the branch